### ISSUE TITLE
feedback

### ISSUE THREAD

**Issue by nomisum** [gruppe-adler/TvT_HuntingHoppers.tem_anizay#1]
- [ ] better armor opfor boss
- [ ] dshkm rearm possibility
- [ ] no mags in respawning vehicles[?]
- [ ] opfor boss shouldnt instantly lose on death, as long as bomb explodes (or MAYBE dead man trigger?)
- [ ] ticket system for lifes?

**Comment by Fusselwurm** [comment]
>  opfor boss shouldnt instantly lose on death, as long as bomb explodes (or MAYBE dead man trigger?)

do you want the opfor boss to respawn? what would be the victory condition for blufor then? I see how the balancing *might* need to be changed, but it seems to me we need minor adjustments, not adding complexity to the win conditions.

**Comment by nomisum** [comment]
> do you want the opfor boss to respawn? what would be the victory condition for blufor then? I see how the balancing _might_ need to be changed, but it seems to me we need minor adjustments, not adding complexity to the win conditions.

would shift dynamics from "kill the boss" to "defuse", which is harder for blufor to achieve. a VIP in heavy fire might again and again lead to sudden/unexpected/unearned wins.

i feel like the shift to 7min time to detonation changed dynamics already quite a lot to successful ground attacks.

**Comment by Fusselwurm** [comment]
Hmmm why then have a boss at all :thinking: we might as well spawn a bunch of satchel charges at the start (with special detonators that allow detonation only after X time or whatever) instead and let opfor figure out for themselves where and what they mean to bomb. 

otoh that leads to problems when people start to parallel-bomb multiple locations. :sweat:

**Comment by nomisum** [comment]
yes, having the ability locked to one person gives blufor the possibility to concentrate and opfor the necessity to concentrate on specific locations, which we want for highest firefight chances
